cypress/support/utils/utils.js

```diff
@@ -82,23 +81,20 @@ const appendHumanReadableNow = str => {
 };
 
 const getLanguageSpecificWorkaround_date = new Date();
+let getLanguageSpecificWorkaround_waitingLock = false; // used so that we don't try to sleep twice
 const getLanguageSpecificWorkaround = () => {
-  const TIME_TO_WAIT = 6 * 1000;
+  const TIME_TO_WAIT = 10 * 1000;
   return cy.get('body').then(function() {
```

Comment:
can't do better (due to lack of knowledge and maybe coz there's nothing better to do?). 

_related: It would help me to me able to use java's `synchronized` block for this function, but JS doesn't have that `:V(`._